thunx-spring/src/main/java/eu/xenit/contentcloud/thunx/spring/data/rest/AbacConfiguration.java

```diff
@@ -80,17 +79,20 @@ public Object postProcessAfterInitialization(Object bean, String beanName) throw
                     var invokerFactory = applicationContext.getBean(RepositoryInvokerFactory.class);
                     var resourceMetadataResolver = applicationContext.getBean(ResourceMetadataHandlerMethodArgumentResolver.class);
                     var factory = applicationContext.getBean(QuerydslBindingsFactory.class);
```

Comment:
Not your change, but better var name would be appropriate, like: "querydslBindingsFactory` or something